Gemfile

```diff
@@ -20,3 +20,5 @@ gem "rqrcode"
 gem "rake"
 gem "minitest"
 gem "rack-test"
+
+gem 'rubeepass', '~> 2.0', '>= 2.0.8'
```

Comment:
Please wrap this in a 

    group :keepass, :optional => true

That way this dependency won't be imposed on everyone unless they need this specific importer.

You can also add a section to `README.md` and mention that the user has to do `bundle install --with keepass` to use it.